### ISSUE TITLE
docker and x509 error

### ISSUE THREAD

**Issue by rastarr** [stashapp/stash#4085]
I'm a new contributor for a week or two. Having cleaned up and contributed quite a few, I've got 1 remaining.

I'm running the latest docker version on a Synology NAS but when scraping a few sites like Hustler and PornWorld, I'm getting this error -

```
023-09-04 09:55:55
Error   
scrapeSingleScene: input: scrapeSingleScene scraper pornworld: Get "https://pornworld.com/autocomplete?query=Porn World - 2023-01-26 - Neighborhood Studs Plug Billie Star's Holes in BBC DP Gp2565 [WEBDL-720p].mp4": x509: certificate has expired or is not yet valid: current time 2023-09-04T09:55:55+07:00 is after 2021-10-26T04:18:29Z
```

The Synology has a valid certificate so I can't understand why the x509 error is coming up. Is this an image thing that needs to be updated or is there a docker config I have missed? (I'm using the docker-compose from this repo)

Any help to fix this would be most welcome.

**Comment by feederbox826** [comment]
Was collecting docker-related issues and saw this, This seems to be an SSL certificate error on the side of pornworld

and looking at their certificate history https://crt.sh/?q=pornworld.com it seems like they had some mix-up with SSL certificates, this is unlikely to be an issue with stash

**Comment by rastarr** [comment]
I was more hoping for feedback on how to trackdown down why it's happening to my installation of stash

**Comment by feederbox826** [comment]
I don't believe it's an error on your side, you would have to disable certificate checks.
